Off-Top-App/off-top-flutter
# OT-CreateGraph | Created a reports graph on the reports page to show user data

**What I did:** 
-I created a graph on the Reports tab in the nav bar. The graph shows some static data for your heartbeat and offtop score.

**How to test:** 
-Most of you already know this but you will need Zookeeper, Kafka, Spring-boot, and port forwarding running to log into app.
-Once you are able to log into the app, you just need to click on the reports tab and you will see the graph generate. If you click onto another tab and go back to the reports, it will generate again.
